src/main/java/org/folio/ld/dictionary/model/Resource.java

```diff
@@ -29,7 +29,9 @@ public class Resource {
 
   private FolioMetadata folioMetadata;
 
-  private Date updatedAt;
+  private Date createdDate;
+
+  private Date updatedDate;
```

Comment:
Shouldn't we make the corresponding change in 'mod-linked-data'?

Reply:
yes PK, it's the reason why MODLD-603 not working in dev env. I included fix in this PR.